solr/core/src/test/org/apache/solr/request/TestRemoteStreaming.java

```diff
@@ -48,14 +48,17 @@ public class TestRemoteStreaming extends SolrJettyTestBase {
 
   @BeforeClass
   public static void beforeTest() throws Exception {
+    System.setProperty("solr.enableRemoteStreaming", "true");
```

Comment:
Should add a new test case here to verify streaming fails when sysprop is false.